src/renderer/components/publication/TagManager.tsx

```diff
@@ -107,7 +120,9 @@ export class TagManager extends React.Component<Props, TagManagerState> {
     }
 
     private handleChangeName(e: any) {
-        this.setState({ nameNewTag: e.target.value });
+        const currentTag: string = e.target.value.trim();
```

Comment:
This line of code prevents adding tags with whitespace in the middle. 

Anyway, as I said in the comment above, let's address whitespace normalization in another PR please. :)